java/de/yessoft/android/mohammad/services/MessageReceiver.java

```diff
@@ -0,0 +1,60 @@
+package de.yessoft.android.mohammad.services;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.media.RingtoneManager;
+import android.net.Uri;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import de.yessoft.android.mohammad.R;
+import de.yessoft.android.mohammad.ui.MainActivity.MainActivity;
+
+public class MessageReceiver extends FirebaseMessagingService {
+    // private static final String TAG = "ML_MessageReceiver";
+
+    private static final int REQUEST_CODE = 1;
```

Comment:
what is the meaning of those numbers , could you please add some documentation

Reply:
The good sullotion would be to create error handler view, and send it an error code which would be translated to an error message, using Log or a View, But the time limit didn't allow such a thing